contracts/interfaces/RevokeAndPublish.sol

```diff
@@ -0,0 +1,32 @@
+pragma solidity 0.4.19;
+import '../EthereumClaimsRegistry.sol';
+
+
+/// @title Revoke and Publish - an interface for publishing data and 
+///        rotating access to publish new data
+contract RevokeAndPublish {
+
+    mapping(address => address) public manager;
+    EthereumClaimsRegistry registry = EthereumClaimsRegistry(0xAcA1BCd8D0f5A9BFC95aFF331Da4c250CD9ac2Da);
```

Comment:
Are we really hardcoding the Claims registry address here? Is it because we can deploy it with the same address on any chain as was suggested in the ERC?

Reply:
Yes! You can take a look in the readme and `scripts/generateDeployTxs.js`